lib/tasks/auth_tokens.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+namespace :auth_tokens do
+  desc "Verify each authorized AuthToken that it's still authorized"
+  task :reverify_authorized, %i[count start] => :environment do |_task, args|
+    exit if ENV["READ_ONLY"].present?
+    args.with_defaults(count: 500, start: nil)
+    last_id = "none"
+
+    begin
+      AuthToken
+        .authorized
+        .in_batches(of: args.count, start: args.start)
+        .each do |token_batch|
+          token_batch.each do |token|
```

Comment:
You could get rid of the double `.each` by using [find_each](https://apidock.com/rails/ActiveRecord/Batches/find_each) instead of `in_batches`. It should accept the same parameters and is basically syntactic sugar for what you have here.